src/lib.rs

```diff
@@ -167,19 +175,35 @@ impl Config {
         println!("cargo:rustc-link-search=native={}", dst.display());
         println!("cargo:rustc-link-lib=static={}",
                  &output[3..output.len() - 2]);
+
+        // Add specific C++ libraries, if enabled.
+        if self.cpp {
+            println!("cargo:rustc-link-lib=static=stdc++");
```

Comment:
Actually it doesn't compile if I omit this line, so I suppose rustc doesn't link the C++ standard library by default. :p